### PR TITLE
Feat:  Library page

### PR DESCRIPTION
**Pull request by kmaid**

### CODE REVIEW

File: packages/react-components/src/organisms/__tests__/LibraryCard.test.tsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    <LibraryCard
+      {...libraryCardProps}
+      teamHref={'/network/teams/123'}
+      team={{
+        id: '123',
+        displayName: 'Team A',
+      }}
+    />,
+  );
+  const link = getByText('Team A') as HTMLAnchorElement;
```
Comment (jeysal) [inline packages/react-components/src/organisms/__tests__/LibraryCard.test.tsx:59]:
```suggestion
  const link = getByText('Team A', { selector: 'a' }) as HTMLAnchorElement;
```
could make it safe I think

File: packages/react-components/src/templates/__tests__/LibraryPageHeader.test.tsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+it('renders the header', () => {
+  const { getByRole } = render(<LibraryPageHeader {...props} />);
+  expect(getByRole('heading')).toBeVisible();
+});
+
+it('Passes query correctly', () => {
+  const { getByRole } = render(
+    <LibraryPageHeader {...props} query={'test123'} />,
+  );
+  expect((getByRole('textbox') as HTMLInputElement).value).toEqual('test123');
```
Comment (jeysal) [inline packages/react-components/src/templates/__tests__/LibraryPageHeader.test.tsx:18]:
```suggestion
  expect(getByRole('textbox')).toHaveValue('test123');
```

**Review by jeysal — CHANGES_REQUESTED** [review]

File: packages/model/src/research-output.ts [hunk 3]
```diff
@@ -32,4 +34,5 @@ export type ResearchOutputResponse = Omit<
 > & {
   readonly id: string;
   readonly created: string;
+  readonly team?: Pick<TeamResponse, 'id' | 'displayName'>;
```
[the remark this answers is not in the document]
Comment (kmaid) [reply packages/model/src/research-output.ts:37]: I copied this from his PR

File: packages/react-components/src/organisms/LibraryCard.tsx [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+  paddingRight: `${15 / perRem}em`,
+});
+
+const typeStyles = css({
+  display: 'flex',
+  alignItems: 'center',
+  textTransform: 'capitalize',
+});
+
+const tagStyles = css({ padding: `0 ${2 / perRem}em` });
```
Comment (jeysal) [inline packages/react-components/src/organisms/LibraryCard.tsx:63]: This came from design QA? Should it not be applied to all `TagLabel`s?

**Review by jeysal — APPROVED** [review]
LGTM except for weird `TagLabel` styling exception, I hope we can avoid that talking to James? :smile:

**Pull request merged**